layout/hud/cgaz.xml

```diff
@@ -8,9 +8,9 @@
 		<include src="file://{resources}/scripts/hud/cgaz.js" />
 	</scripts>
 
-	<MomHudCgaz class="full-width full-height">
+	<MomHudCgaz class="full-width full-height" alternateTicks="false">
 		<Panel id="AccelContainer" class="cgaz__container">
-			<ConVarEnabler class="cgaz" convar="mom_df_hud_accel_enable" togglevisibility="true">
```

Comment:
If we're removing this class we should probably rethink `cgaz.scss` a bit. We could only have the `cgaz__container` class, but I'm inclined to say we just remove it and use `full-width` and `horizontal-align-left` instead of the `cgaz__container` class.

Reply:
Ok, can we make an issue for this and review with the rest of the upcoming BetterCgaz(tm) changes?

Reply:
#31 captures this comment